apiron/client.py

```diff
@@ -108,10 +109,20 @@ def build_request_object(
         headers=None,
         cookies=None,
         auth=None,
+        **kwargs
     ):
         host = cls.choose_host(service=service)
 
+        if path_kwargs:
+            warnings.warn(
+                'path_kwargs is no longer necessary and will be removed in a future version of apiron. '
+                'You can call endpoints using plain keyword arguments instead!',
+                RuntimeWarning,
+                stacklevel=4
+            )
```

Comment:
I would have preferred this to be a `DeprecationWarning` but since this is dynamic, the lines where it's used would neither get highlighted statically in PyCharm nor produce this log at runtime.